### ISSUE TITLE
Fix ESLint config setup & adapt it to the current state of Nova

### ISSUE THREAD

**Issue by xavcz** [TelescopeJS/Telescope#1470]
This is a [starter issue](http://hood.ie/blog/starter-issues.html), _highly inspired_ by the great job of the folks at Hoodie. 👍
### 🎃🙀👕🔭😻 Hacktoberfest: Trick or Treat!

If you haven’t yet, sign up for [Hacktoberfest](https://hacktoberfest.digitalocean.com/) to earn an exclusive T-Shirt. 

We are sure you can learn a cool trick or two in the process on how to hack & customize Telescope Nova! 🔭
### 🤔 What you will need to know

Meteor, React, ESLint rules.
### 🎯 The Goal

> If the style of every bit of code is consistent, that ensures that bits of code that look the same actually are the same - there isn’t any hidden punctuation or gotchas that you don’t expect, **so you can focus on understanding the logic instead of the symbols.**

Let's give more consistency to Nova codebase! The current setup of [ESLint](http://eslint.org/) is broken, and not really up to date.

That would be great to have the right plugins installed as _dev dependencies_. As of now, we would tend to ES6/7 syntax with some relaxed rules.

We could start by following AirBnB / React / Meteor rules and then disable some of them (allow no spaces after curly-braces, allow no dangling commas, ..).

Some links as a starting point:
- [Meteor recommendations on how to install and run ESLint](https://guide.meteor.com/code-style.html#eslint-installing)
- [Article on linting Meteor apps by the creator of `eslint-plugin-meteor`](https://medium.com/@dferber90/linting-meteor-8f229ebc7942)
### 📋 Step by Step

**Working on your first Pull Request?** You can learn how from this _free_ series [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).

---
- [x] Claim this issue (@moimikey)
- [x] Fork the repo, [setup repository locally](https://github.com/TelescopeJS/Telescope#getting-started) & checkout on `devel` branch.
- [x] Install the right ESLint plugins as dev dependencies
- [x] Configure the rules to fit with the state of Nova (Meteor + React, kind of relax)
- [x] Lint the project and fix some surface issues 
- [x] Open a PR to `devel` branch, mentioning this issue.
- [x] Someone else: review & merge! 🎉

Ping us here, in the [Telescope Slack room](http://slack.telescopeapp.org) or on [Twitter](https://twitter.com/telescopeapp) if you have any questions 😉

**Comment by moimikey** [comment]
i'd be down :o

**Comment by xavcz** [comment]
Yeah sure! Go for it! Don't hesitate to drop by the [Slack Room](https://telescopeapp.slack.com/messages/) to chat about the code style if you want!

**Comment by moimikey** [comment]
SO, @xavcz, Meteor is pretty hardcore with their preset. this was the output from `eslint`:

```
✖ 4122 problems (3102 errors, 1020 warnings)
```

with the following rules:

```
  "rules": {
    "babel/generator-star-spacing": 0,
    "babel/new-cap": 1,
    "babel/array-bracket-spacing": 0,
    "babel/object-curly-spacing": 0,
    "babel/object-shorthand": 0,
    "babel/arrow-parens": 0,
    "babel/no-await-in-loop": 1,
    "comma-dangle": 0,
    "key-spacing": 0,
    "no-extra-boolean-cast": 0,
    "no-undef": 1,
    "no-unused-vars": 1,
    "no-console": 1
  }
```

`eslint:recommended` is less crazy and more manageable to fix:

```
✖ 878 problems (18 errors, 860 warnings)
```

obviously depending on how strict/un-strict. I can of course categorize the errors of either and turn off whatever is more prevalent. there are a few ways to go on about it. There's an accompanying commit so far with the `eslint:recommended` preset enabled.

**Comment by xavcz** [comment]
Ouch, yeah that's hardcore haha! 😄 

That's really to move forward with the lint config, thanks @moimikey! 🚀  

I've submitted a review on your PR with some feedbacks on Meteor coupled with ESLint, things I've gathered from other repos and docs. Your insights on these are more than welcomed!! 👍

**Comment by xavcz** [comment]
fixed by #1474  and https://github.com/TelescopeJS/Telescope/commit/4ae1c01eecb406a16218a21915c7735a2d40960f 🎉
